js/main.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var listsDescription = {
+  typeList: ['palace', 'flat', 'house', 'bungalo'],
```

Comment:
для констант нужен верхний регистр
```suggestion
  TYPELIST: ['palace', 'flat', 'house', 'bungalo'],
```